src/routes/concepts/stores.mdx

```diff
@@ -104,26 +167,37 @@ setStore("users", (otherUsers) => [
 ])
 ```
 
-    <Callout>
-      Separating read and write capabilities to a store presents a valuable debugging advantage.
+We could also set update the value of `userCount` whenever a new user is added.
```

Comment:
This sentence feels out of place, could you add more context?

Reply:
@LadyBluenotes I forgot to take out "set" when I reworded this which definitely makes it read weird. It is just to preface the next example. There isn't really a purpose other than to provide a different example of how userCount could be automatically updated based on on the users array. 

I primarily included it because the example was there originally and I figured having another example of a different use case would be good.

```suggestion
The value of `userCount` could also be automatically updated whenever a new user is added to keep it up to date:
```

Let me know what you think about this instead, I don't really know what additional context to add.